.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
 cache:
   directories:
-      - $HOME/.m2/repository
+    - "$HOME/.m2/repository"
+
+install: mvn install -DskipTests=true -Dmaven.javadoc.skip=true
+script:
+  - jdk_switcher use oraclejdk8
+  - mvn test
+  - jdk_switcher use oraclejdk7
```

Comment:
@salimane : we were talking about leveraging some Java 8 features, so were talking about dropping support for JDK7 for compiling.  But no specific plan yet, so we can change this test later if we decide to do that.